test/tableau/extensions/post_extension_test.exs

```diff
@@ -163,6 +163,55 @@ defmodule Tableau.PostExtensionTest do
       assert Blog.PostLayout in vertices
     end
 
+    test "drafts: true will not render a post", %{tmp_dir: dir, token: token} do
+      File.write(Path.join(dir, "my-draft-post.md"), """
+      ---
+      layout: Blog.PostLayout
+      title: My Draft Post
+      date: 2020-01-01
+      categories: post
+      permalink: /post/2020/01/01/my-draft-post/
+      draft: true
+      ---
+
+      Do androids dream of electric sheep?
+      """)
+
+      assert {:ok, config} = PostExtension.config(%{dir: dir, enabled: true})
+
+      token = put_in(token.extensions.posts.config, config)
+
+      assert {:ok, token} = PostExtension.run(token)
+
+      assert %{
+               posts: []
+             } = token
+    end
+
+    test "files in config.drafts will not render a post", %{tmp_dir: dir, token: token} do
+      File.write(Path.join("_drafts", "my-post-in-drafts-folder.md"), """
```

Comment:
use `File.write!/2` in cases where you don't match on the result so the test fails if it fails